src/test/scala/com/cognite/spark/datasource/EventsRelationTest.scala

```diff
@@ -111,9 +112,9 @@ class EventsRelationTest extends FlatSpec with Matchers with SparkTest {
       .option("metricsPrefix", metricsPrefix)
       .load()
       .where(s"startTime > 1584601199999")
-    assert(df.count == 3)
+    assert(df.count == 2)
     val eventsRead = getNumberOfRowsRead(metricsPrefix, "events")
-    assert(eventsRead == 3)
+    assert(eventsRead == 2)
```

Comment:
Why does this change?

Reply:
Excellent question, I did no changes to Events in this PR so some test data must have changed.

Reply:
Looks like a few of these tests were actually reading test data created by other tests, and so were based on them having completed successfully on the previous run. I'll change them to not do that.